README.md

```diff
@@ -8,6 +8,12 @@ Workspace for a Rust wrapper for SuiteSparse:GraphBLAS. The workspace contains t
 ## Dependencies
 graphblas_sparse_linear_algebra uses the [SuiteSparse:GraphBLAS](https://github.com/DrTimothyAldenDavis/GraphBLAS) GraphBLAS implementation developed by Timothy A. Davis.
 
+To build and test on Ubuntu 24.04, [install Rust](https://www.rust-lang.org/tools/install) and ensure the dependencies are installed:
+
+```
+sudo apt install build-essential pkg-config libssl-dev cmake clang
```

Comment:
Definitely helpful, ran into this problem myself too. Did you check this is complete? For example, git will be included already, right?

In general, this hint is best placed in the  [readme](https://github.com/code-sam/graphblas_sparse_linear_algebra/blob/main/suitesparse_graphblas_sys/README.md) of [suitesparse_graphblas_sys](https://github.com/code-sam/graphblas_sparse_linear_algebra/tree/main/suitesparse_graphblas_sys). 

Instead of these specific build instructions, this readme in the repository root would benefit from a better reference to the build instructions for suitesparse_graphblas_sys.

Reply:
I’ve added `git`. The list should be complete now: these are the packages that I needed to install on a quite fresh vanilla Ubuntu 24.04 installation.

I’ve moved the hint and added the reference as you suggested.